onnxruntime/core/session/onnxruntime_c_api.cc

```diff
@@ -2725,8 +2725,15 @@ static constexpr OrtApi ort_api_1_to_16 = {
     &OrtApis::GetOptionalContainedTypeInfo,
     &OrtApis::GetResizedStringTensorElementBuffer,
     &OrtApis::KernelContext_GetAllocator,
-    &OrtApis::GetBuildInfoString};
-// End of Version 15 - DO NOT MODIFY ABOVE (see above text for more information)
+    &OrtApis::GetBuildInfoString,
+    // End of Version 15 - DO NOT MODIFY ABOVE (see above text for more information)
+
+    // Start of Version 16 API in progress, safe to modify/rename/rearrange until we ship
+    &OrtApis::CreateROCMProviderOptions,
+    &OrtApis::UpdateROCMProviderOptions,
+    &OrtApis::GetROCMProviderOptionsAsString,
+    &OrtApis::ReleaseROCMProviderOptions,
+};
```

Comment:
This may cause lint issue. No trailing comma and the closing bracket on the same line.

I found it helpful to run lintrunner locally.

Reply:
No lint issue found. Our CI is also happy with this form. I will stick with it as it minimizes the diff change.